### PR TITLE
feat: [IOAPPCIT-41,IABT-1427,IABT-1428] Adds Zendesk permission label and distincted flow for Add Card issue

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR implements a new logic to properly handle assistance request for card or payment

## List of changes proposed in this pull request
- Updated Navigation Params
- Implemented specific permission item on list
- Add an IP Address specification label

## How to test
Try to add a Payment Method, generate the failure, ask for assistance on the add method failure list

https://user-images.githubusercontent.com/3959405/215705441-b0d3c552-b3ed-4b3d-bb2b-e080bb5d98b9.mp4

### CODE REVIEW

File: ts/features/zendesk/screens/ZendeskAskPermissions.tsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ const getItems = (props: ItemProps): ReadonlyArray<ItemPermissionProps> => [
   },
   {
     id: "profileFiscalCode",
-    icon: <FiscalCodeIcon {...iconProps} />,
+    icon: <CardIcon {...iconProps} />,
```
Comment (hevelius) [inline ts/features/zendesk/screens/ZendeskAskPermissions.tsx:103]: So the icon for FiscalCode and CardIssues is the same?

Comment (CrisTofani) [reply ts/features/zendesk/screens/ZendeskAskPermissions.tsx:103]: @hevelius yes according to [figma](https://www.figma.com/file/PsDwmBK5bqKZdXsycrTolb/Flusso-Assistenza-(Zendesk)?node-id=12%3A1798&t=3hzGNYnSQd7svFW9-4)

**Review by hevelius — COMMENTED** [review]
LGTM

File: locales/en/index.yml [hunk 2]
```diff
@@ -3242,13 +3242,16 @@ support:
...
+    ipAddressValue: "Including the assumed geographic location"
     appVersionsHistory: "App version history"
     appVersionsHistoryValue: "To determine if the issue is due to the most recent update"
     identityProvider: "Identity Provider"
     navigationData: "In-app navigation data"
     navigationDataValue: "Such as the the last screen or message displayed in IO."
     stock: "Payment Notice details"
     stockValue: "Including the error you just received"
+    card: "Payment methods details"
+    cardValue: "Number (encrypted), expiration, circuit"
```
Comment (thisisjp) [inline locales/en/index.yml:3254]:
```suggestion
    cardValue: "Number (encrypted), expiration, network"
```

**Review by thisisjp — APPROVED** [review]

**Review by hevelius — APPROVED** [review]
LGTM

**Pull request merged**